### PR TITLE
Created fake data and a function that will build data in database 

### LINKED ISSUE #19: Create fake data
- We need JSON file to storing the fake data and another file to storing all the features with icon's URL that the website will use it.

### LINKED ISSUE #37: [TypeScript] create a function to build the initial data in database
In this task I will use bulkCreate() method to create our database once run a specific script

### PR DESCRIPTION
**Pull request by tariqabusada**
- Wrote some of data that will be shown on the web site as an initial data in each customer request state.
- created 'buildFakeData' function to build the data where in 'fakeData.json' file
- wrote a script to running the build file.

### CODE REVIEW

**Review by mai-mohammed — APPROVED** [review]

**Review by husamkamal — APPROVED** [review]

**Review by LinaYahya — COMMENTED** [review]
Create a script to run function that will inject fakeData to our db

**Comment by tariqabusada** [comment]
> Create a script to run function that will inject fakeData to our db

Done, check it

**Review by abdou059 — APPROVED** [review]

File: server/package.json [hunk 1]
```diff
@@ -44,6 +46,7 @@
     "socket.io": "^4.5.2",
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
-    "yup": "^0.32.11"
+    "yup": "^0.32.11",
+    "typescript": "^4.8.4"
```
Comment (LinaYahya) [inline server/package.json]: remove to devDependancy

**Review by LinaYahya — COMMENTED** [review]
Great Job @tariqabusada 
Really Great!

**Pull request merged**